docs/journey/day1/README.md

```diff
@@ -33,9 +33,11 @@ $ kam bootstrap \
   --image-repo quay.io/<username>/<image-repo> \
   --dockercfgjson ~/Downloads/<username>-auth.json \
   --git-host-access-token <your git access token> \
-  --output <path to write GitOps resources>
+  --output <path to write GitOps resources> \
   --push-to-git=true
 ```
+**NOTE**: Flag `--push-to-git=true` brings the bootstrapped environment up, uses SSH protocol to communicate with Git server then create and push bootstrap files into the gitops repo. You need to make sure that public ssh key is added into your GitHub account when using `--push-to-git=true` flag.
```

Comment:
It's not really bringing the bootstrapped environment "up" as such, it's pushing it to a Git hosting provider.

Maybe we should say something like...

> When you choose to push the generated resources to your GitOps repository, this will execute `git` locally on the developer machine, which will in turn authenticate the push using your local SSH keys, this means that you need to be able to push to a Git repository from your local machine.

Does that capture it better?

Reply:
It looks fine but I think it is worth mentioning that user needs to add a public key to github/gitlab account.   We could add some reference to  https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/connecting-to-github-with-ssh..